crates/replace/src/lib.rs

```diff
@@ -183,15 +216,16 @@ decl_module! {
         /// # Arguments
         ///
         /// * `origin` - the initiator of the transaction: the new vault
-        /// * `replace_id` - the unique identifier for the specific request
+        /// * `old_vault` - id of the old vault that we are partially replacing
         /// * `collateral` - the collateral for replacement
+        /// * `btc_address` - the address that old-vault should transfer the btc to
```

Comment:
I don't think it's worth the extra load - all it would do is save one RPC call to bitcoind; there's no logic in the client except for converting the generated address to a strongly typed one.